flagsmith-jira-app/src/jira.ts

```diff
@@ -70,6 +70,10 @@ const setEntityProperty = async <T>(
     method: "PUT",
     body: JSON.stringify(value),
     jsonResponse: false,
+    headers: {
+      Accept: 'application/json',
+      "Content-Type": 'application/json'
```

Comment:
```suggestion
      "Accept": "application/json",
      "Content-Type": "application/json",
```

Reply:
@matthewelwell I have addressed this. I am not familiar with github actions, so not sure where exactly I can find the build status for my commit.